### PR TITLE
Update i18n-ai-translate, use ChatGPT

### PR DESCRIPTION
**Pull request by taahamahdi**

### CODE REVIEW

File: src/ci_checks/i18n-ai-translate.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ cp i18n/en.json i18n/en-latest.json
 git checkout $merge_base -- i18n/en.json
 if ! diff i18n/en-latest.json i18n/en.json; then
     echo "Updating translations..."
-    npx i18n-ai-translate diff -b i18n/en.json -a i18n/en-latest.json -l "English" --verbose
+    npx i18n-ai-translate diff -b i18n/en.json -a i18n/en-latest.json -l "English" --verbose --engine chatgpt
```
Comment (Brainicism) [inline src/ci_checks/i18n-ai-translate.sh:10]: does this default to gpt3.5 or 4?

Comment (Brainicism) [reply src/ci_checks/i18n-ai-translate.sh:10]: Don’t think this is something that should have a default, especially if both are available to users and have different costs

Comment (taahamahdi) [reply src/ci_checks/i18n-ai-translate.sh:10]: Is it something that should be fixed on the library side or something here

Comment (Brainicism) [reply src/ci_checks/i18n-ai-translate.sh:10]: library, but a result, here as well

**Review by Brainicism — APPROVED** [review]

**Pull request merged**